python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -1454,19 +1454,22 @@ class CompareWarpAssembleCoaddConfig(AssembleCoaddConfig):
         dtype=int,
         default=2
     )
-    spatialThreshold = pexConfig.Field(
+    spatialThreshold = pexConfig.RangeField(
         doc="Unitless fraction of pixels defining how much of the outlier region has to meet the "
             "temporal criteria",
         dtype=float,
-        default=0.5
+        default=0.5,
+        min=0., max=1.
     )
 
     def setDefaults(self):
         AssembleCoaddConfig.setDefaults(self)
+        self.statistic = 'MEAN'
         self.assembleStaticSkyModel.warpType = 'psfMatched'
-        self.assembleStaticSkyModel.statistic = 'MEDIAN'
+        self.assembleStaticSkyModel.statistic = 'MEANCLIP'
+        self.assembleStaticSkyModel.sigmaClip = 1.5
```

Comment:
Why such harsh clipping?

Reply:
Borrowed from SafeClip. Running a grid of configs is still a to do: DM-12698

Reply:
But harsh clipping like this is part of the SafeClip algorithm (we deliberately toss out anything that looks even remotely bad in order to make a very clean model of the sky), whereas I wouldn't have thought it would be for this.

Reply:
This is the same idea.  Make a very very clean and naive robust PSF-match coadd to serve as the naive model of the static sky.

Reply:
Ah, I see --- this is for the `assembleStaticSkyModel`. Fair enough, then.

Reply:
For 2< N<=5 especially, we have to be strict to get the junk out of the naive template:

```
import  lsst.afw.math as afwMath
arr = [1,10,10,20]
sctrl = afwMath.StatisticsControl()
sctrl.setNumSigmaClip(1.5)
sctrl.setNumIter(3)
print afwMath.makeStatistics(arr, afwMath.MEANCLIP, sctrl).getValue()
print afwMath.makeStatistics(arr, afwMath.STDEV, sctrl).getValue()
> 10.0
> 7.76208734813
```

Reply:
Perhaps that's due to a shortcoming in our statistics suite (no robust stdev), or because you're not using `STDEVCLIP`:
```
>>> ss = afwMath.makeStatistics(arr, afwMath.MEANCLIP | afwMath.STDEVCLIP | afwMath.NCLIPPED, sctrl)
>>> ss.getValue(afwMath.MEANCLIP)
10.0
>>> ss.getValue(afwMath.STDEVCLIP)
0.0
>>> ss.getValue(afwMath.NCLIPPED)
2.0
```

(`NCLIPPED` is a feature being implemented on DM-9953.)

Reply:
That's cool.  My point was that if I increase `sctrl.setNumSigmaClip(1.5)` to  `sctrl.setNumSigmaClip(2.6)` the 2 outliers don't get clipped anymore.
```
>>> sctrl.setNumSigmaClip(2.6)
>>> print afwMath.makeStatistics(arr, afwMath.MEANCLIP, sctrl).getValue()
10.25
```

Reply:
OK. How common is it that 40% of the inputs have outliers?